policyengine_us/parameters/gov/states/nd/tax/income/credits/marriage_penalty/maximum.yaml

```diff
@@ -2,13 +2,20 @@ description: North Dakota has a nonrefundable marriage-penalty credit for joint
 values:
   2021-01-01: 201
   2022-01-01: 208
+  2023-01-01: 287
 
 metadata:
   label: marriage-penalty credit maximum amount
   unit: USD
+  uprating: gov.irs.uprating
   period: year
   reference:
   - title: 2021 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
-    href: https://revenue.nebraska.gov/files/doc/2021_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf#page=16
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2021-iit/2021-individual-income-tax-booklet.pdf#page=16
   - title: 2022 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
-    href: https://revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf#page=16
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2022-iit/2022-individual-income-tax-booklet.pdf#page=16
+  - title: 2023 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2023-iit/2023-individual-income-tax-booklet.pdf#page=16
+  - title:  North Dakota legal code Section 57-38-01.28
```

Comment:
still need legal code here

Reply:
I'm working on these now. Will fix them by the end of today.